### ISSUE TITLE
Failing CI Builds

### ISSUE THREAD

**Issue by stevehipwell** [DefinitelyTyped/DefinitelyTyped#16692]
Just a heads up that the CI build is currently failing with the following issues:

```
Warning: ts.forEachLeadingCommentRange is not a function
Warning: ts.isWhiteSpaceLike is not a function
ts.forEachLeadingCommentRange is not a function
```

I've tried to run the build directly from master with the same outcome. This issue is able to be seen on the following [pull request](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16382).

**Comment by ammuench** [comment]
Having the same issue on [my latest PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16734) as well, can't figure out how to resolve

**Comment by WrathOfZombies** [comment]
+1 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16741

**Comment by andy-ms** [comment]
As mentioned in #16741, this is a dtslint bug. You can fix it by adding a `// TypeScript Version: 2.1` comment at the end of the header in `index.d.ts` (usually line 5). See Microsoft/dtslint#39.

**Comment by mhegazy** [comment]
This is fixed by https://github.com/Microsoft/dtslint/pull/40. please rebase your PRs to master.
